src/API/Endpoints/Reports/PaymentMethods.php

```diff
@@ -18,7 +18,12 @@ public function get_report( $request ) {
 
 		// Use give_count_payments logic to get payments
 		$gateways = give_get_payment_gateways();
-		$stats    = new \Give_Payment_Stats();
+
+		if ( $this->testMode === false ) {
```

Comment:
@henryholtgeerts As I can see `testMode` will get string param from rest API. did you cast it somewhere in code because comparing boolean to the string will not give desire result?

Reply:
Yes, it was previously casted with filter_var, but now with boolval in the sanitize_test_mode function. Ref: https://github.com/impress-org/givewp/commit/1dbdf735ba097a7e5d594103f0dca2a96aaf5203